src/reducers/fullscreenGrid.tsx

```diff
@@ -0,0 +1,31 @@
+import { Reducer } from 'redux';
+import { ActionType, createAction, getType } from 'typesafe-actions';
+
+export const actions = {
+  toggleMainSidePanel: createAction('TOGGLE_MAIN_SIDE_PANEL'),
```

Comment:
It seems like it might be worth being able to toggle either panel from the beginning. It wouldn't be much more code than you're adding right now and would mean we wouldn't need those changes later.

Maybe a `togglePanel` action which accepts an argument for which panel to toggle?

Reply:
I added a `toggleAltSidePanel` action so that we don't have to deal with (invalid) panel IDs and `undefined` values.